src/12_scopes.py

```diff
@@ -4,29 +4,28 @@
 # When you use a variable in a function, it's local in scope to the function.
 x = 12
 
-def change_x():
```

Comment:
For this file we shouldn't be changing return values or variables, we should be using `global` and `nonlocal` keywords to achieve the goals.